src/misc/lv_event.c

```diff
@@ -27,6 +27,14 @@
  *  STATIC PROTOTYPES
  **********************/
 
+/* Traverse the list to delete the objects marked for deletion */
+static void cleanup_event_list(lv_event_list_t * list);
+
+inline static void event_mark_deleting(lv_event_list_t * list, lv_event_dsc_t * dsc);
```

Comment:
Please use `static inline` for consistency

Reply:
Without adding `inline`, the compiler automatically inlines short functions.

Reply:
We can be explicit to be sure. :wink: